src/IpFilter.php

```diff
@@ -30,6 +32,9 @@ final class IpFilter implements MiddlewareInterface
      * @psalm-param array<array-key, string> $ipRanges
      */
     public function __construct(
+        /**
+         * @deprecated Will be removed in version 2.0. {@see IpRanges} from `network-utilities` package is used instead.
+         */
         private ValidatorInterface $validator,
```

Comment:
```suggestion
        ValidatorInterface $validator,
```

Reply:
Why?

Reply:
Validator parameter is keeped for BC only.